### PR TITLE
Fix #32 - tunnel chain should set the target host in the 'host' heaer and not in the path parameter

### LINKED ISSUE #32: tunnel chain should set the target host in the 'host' header and not in the path parameter
The handler_tunnel_chain.js file creates a `CONNECT` request to the real proxy and sets the desired target in the path parmater of the request options.
As far as I know the standard it to set it in the 'host' header...

### PR DESCRIPTION
**Pull request by regevbr**
The handler_tunnel_chain.js file creates a CONNECT request to the real proxy and sets the desired target in the path parmater of the request options.
As far as I know the standard it to set it in the 'host' header...

### CODE REVIEW

**Comment by regevbr** [comment]
Validated logic works in production

File: src/handler_tunnel_chain.js [hunk 1]
```diff
@@ -20,12 +20,16 @@ export default class HandlerTunnelChain extends HandlerBase {
...
+
         const options = {
             method: 'CONNECT',
             hostname: this.upstreamProxyUrlParsed.hostname,
             port: this.upstreamProxyUrlParsed.port,
-            path: `${this.trgParsed.hostname}:${this.trgParsed.port}`,
-            headers: {},
+            path: targetHost,
+            headers: {
+                host: targetHost,
```
Comment (jancurn) [inline src/handler_tunnel_chain.js]: This should be `Host`

Comment (regevbr) [reply src/handler_tunnel_chain.js]: I will change though header names are case insensitive so that doesn't really matter

**Comment by regevbr** [comment]
@jancurn sure I wlll do all that

File: src/handler_tunnel_chain.js (hunk 1 above)
Comment (jancurn) [reply src/handler_tunnel_chain.js]: Great, thanks. You're right, but it's typical to send the headers with the right case.

**Comment by regevbr** [comment]
@jancurn done please review

**Comment by regevbr** [comment]
@jancurn no problem! when can we expect a new version with the fix?

**Comment by mtrunkat** [comment]
I have just released it as `0.2.6` with BETA tag.

**Comment by regevbr** [comment]
@jancurn nice! already in production. Many thanks!

**Pull request merged**